src/test/components/sourcesTable/SourcesTable.test.js

```diff
@@ -119,7 +119,7 @@ describe('SourcesTable', () => {
                 expect(wrapper.find(RowWrapper).first().props().className).toEqual(ROW_WRAPPER_CLASSNAME);
                 expect(wrapper.find(LongArrowAltDownIcon)).toHaveLength(1);
                 expect(wrapper.find(ArrowsAltVIcon)).toHaveLength(expectSortableColumns);
-                expect(wrapper.find('img')).toHaveLength(sourcesDataGraphQl.length);
+                expect(wrapper.find('img')).toHaveLength(0);
```

Comment:
We can just remove this expect.

Reply:
It makes sure that there are no icons :D

Reply:
Well, we could likely check if there are no other elements that are not supposed to be there. We should not check that something does not exist if its never actually there.

Reply:
Before this change the images were there. And I think each change should be tested, otherwise you can't be sure that the change worked. Also, we should test that there are no additional elements on the page, however except useless snapshots, there is no easy way to do it.

Reply:
I disagree with this. Tests are not there to keep history but to test the actual code. There are no images in the code ever. Therefore there is no point to test of there are some.

Reply:
`test the actual code.` It tests the actual code.
`There are no images in the code ever.` Without the test, you can't say it surely.
`expect(true).toBe(true)` is tautology, `expect(wrapper.find('img')).toHaveLength(0)` isn't.

Reply:
> expect(wrapper.find('img')).toHaveLength(0) isn't.

It is. Show me a scenario where there is an img tag in the table.

Reply:
Every scenario, where someone adds the tag in the code. Otherwise, we could throw away testing of every static feature.